src/sql/src/parsers/admin_parser.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use snafu::ResultExt;
+use sqlparser::tokenizer::Token;
+
+use crate::ast::{Expr, Function};
+use crate::error::{Result, SyntaxSnafu};
+use crate::parser::ParserContext;
+use crate::statements::admin::Admin;
+use crate::statements::statement::Statement;
+
+/// `admin` extension parser: `admin function(arg1, arg2, ...)`
+/// or `admin function`
+impl<'a> ParserContext<'a> {
+    /// Parse `admin function(arg1, arg2, ...)` or `admin function` statement
+    pub(crate) fn parse_admin_command(&mut self) -> Result<Statement> {
+        let _token = self.parser.next_token();
+
+        let object_name = self.parser.parse_object_name(false).context(SyntaxSnafu)?;
```

Comment:
nit: should we call the `canonicalize_object_name` here?

Reply:
It's the function name, looks like it doesn't need to call `canonicalize_object_name`.